package/CHANGELOG

```diff
@@ -19,7 +19,7 @@ The rules for this file:
   * 0.18.1
 
 Enhancements
-
+  * Added box/axis/plane centering trajectory transformations
```

Comment:
Instead of adding this line to the changelog, you can update the other line about centering method by listing this PR next to the other one.